Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test
+test:
+	go test -v ./...
+
+.PHONY: build
+build:
+	go build 
+
+.PHONY: lint
+lint: 
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b $(GOPATH_DIR)/bin v1.39.0
```

Comment:
@alenkacz I had to add `sudo` here otherwise it was giving [this](https://github.com/kubernetes-sigs/depstat/runs/2408696149#step:3:6) error. Is it okay to add `sudo` to fix this?

Reply:
I think you don't have to move it to gopath, just download it to ./bin and add .bin to gitignore then run the binary like this `./bin/golangci-lint`

Reply:
Got it! Thanks :)
PTAL now.